app/components/header.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import logo from "../../public/meuch_logo.svg";
+import Link from "next/link";
+
+export default function Header() {
+  return (
+    <header className="navbar bg-white">
+      <div className="flex-1">
+        <a className="btn btn-ghost text-xl">Meuch</a>
```

Comment:
Remove this <a>

Reply:
Resolved : [remove Meuch blank space](https://github.com/MeuchCNAM/ndi-2023/pull/13/commits/3291d04b621abb86acaa69c543d0b6222d694a3e)